### PR TITLE
Update SCH for 5.5 support

### PR DESCRIPTION
**Pull request by redshadowhero**
I mean, it's 100 mp cost. We don't even do mp-based analysis on SCH.

### CODE REVIEW

**Comment by redshadowhero** [comment]
Depends on #1136; please merge that one first.

**Review by ayyaruq — APPROVED** [review]
LGTM, can merge once the 5.5 patch entry is in, thanks for the PR!

File: src/parser/jobs/sch/index.js [hunk 1]
```diff
@@ -27,6 +27,11 @@ export default new Meta({
 		{user: CONTRIBUTORS.NIV, role: ROLES.DEVELOPER},
 	],
 	changelog: [{
+		date: new Date('2020-4-13'),
```
Comment (ayyaruq) [inline src/parser/jobs/sch/index.js:30]:
```suggestion
		date: new Date('2021-04-13'),
```

**Review by ayyaruq — CHANGES_REQUESTED** [review]
Let's not live in the past.

**Review by ayyaruq — APPROVED** [review]

**Pull request merged**